includes/options/class-amp-options-menu.php

```diff
@@ -318,9 +319,13 @@ public function render_theme_support() {
 						<p><?php esc_html_e( 'Your active theme is known to work well in standard or transitional mode.', 'amp' ); ?></p>
 					</div>
 				<?php endif; ?>
+			<?php elseif ( AMP_Theme_Support::theme_supports_reader_mode() ) : ?>
+				<div class="notice notice-success notice-alt inline">
+					<p><?php echo wp_kses_post( $reader_mode_support ); ?></p>
```

Comment:
`wp_kses_post()` might be overly generous here, how about being specific for the expected translation:
```suggestion
					<p><?php echo wp_kses( $reader_mode_support, [ 'strong' => [] ] ); ?></p>
```

Reply:
Ah, good point.